### PR TITLE
V7.1 autologout: Fixes issue with forced logout even when in cisco network

### PR DESCRIPTION
**Pull request by saurjai3**
Ref: https://github.com/ciscospark/spark-js-sdk/pull/488

### CODE REVIEW

File: packages/node_modules/@ciscospark/plugin-wdm/src/device/device.js [hunk 1]
```diff
@@ -250,6 +250,10 @@ const Device = SparkPlugin.extend({
 
       const timer = setTimeout(() => {
         this.spark.request({
+          headers: {
```
Comment (ianwremmel) [inline packages/node_modules/@ciscospark/plugin-wdm/src/device/device.js]: I think changes went in that mean you shouldn't need to explicitly omit these headers. try running your tests without this block.

Comment (saurjai3) [inline packages/node_modules/@ciscospark/plugin-wdm/src/device/device.js]: It seems to run the test cases after removing this block. Hence removing these checks.

**Comment by ianwremmel** [comment]
can this be closed?

**Comment by saurjai3** [comment]
Merged and closed with https://github.com/ciscospark/spark-js-sdk/pull/585

**Pull request closed without merging**